webpack/assets/javascripts/react_app/components/LoginPage/LoginPage.scss

```diff
@@ -14,6 +14,25 @@ $background_image: linear-gradient(
 #login-page {
   height: 100%;
 
+  #login-caption {
+    position: fixed;
+    bottom: 0;
+    padding: 30px;
+    color: white;
+    font-size: 14px;
+
+    .btn-expand {
+      font-size: inherit;
+      padding: 0;
+      margin-left: 10px;
+      color: #52b0fb;
```

Comment:
Can you use a specif value from pf4 variables? or set a new color in `webpack/assets/javascripts/react_app/common/colors.scss` ?